guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -14,6 +14,11 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 
 
+In order to add a layout changes that will be available on every page, a module should use deafult.xml
+
+For Example :
+Layout changes that is added to `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` will be loaded to all the available page.
```

Comment:
We provide information on how to change layout globally but don't mention a possibility to make changes to particular handles. 
I would suggest adding another sentence about editing local handles. i.e. "To add layout changes to a specific page a module should use layout handle that corresponds the page's path. Example: changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page"

Reply:
Also, if we are talking about a module, please replace `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` with something like `app/code/Vendor/Module/view/frontend/layout/default.xml`

Reply:
added the changes